feature_engine/encoding/similarity_encoder.py

```diff
@@ -102,14 +102,22 @@ class StringSimilarityEncoder(CategoricalInitMixin, CategoricalMethodsMixin):
         categories to encode. In this case, similarity variables will be created
         only for those popular categories.
 
-    missing_values : str, default='impute'
+    missing_values: str, default='impute'
         Indicates if missing values should be ignored, raised or imputed. If 'raise' the
         transformer will return an error if the datasets to `fit` or `transform`
         contain missing values. If 'ignore', missing data will be ignored when learning
         parameters or performing the transformation. If 'impute', the transformer will
         replace missing values with an empty string, '', and then return the similarity
         measures.
 
+    keywords: dict, default=None
+        User defined dictionary of keywords, dict(feature: [keyword1, keyword2, ...]).
+        Instead of finding top_k categories in features, encoder will use this keywords
+        to create similarity variables. Useful when someone has domain knowledge of the
+        problem. Could be defined only for several features; in this case for specified
+        features keywords will be used and most common categories will be used for
+        unspecified.
```

Comment:
I don't understand the last sentence of this description. Would you be able to re-phrase it?